guides/common/modules/ref_best-practices-for-sync-plans.adoc

```diff
@@ -0,0 +1,24 @@
+[id="best-practices-for-sync-plans_{context}"]
+= Best practices for sync plans
+
+* Add sync plans to products and regularly synchronize content to avoid overly large changes in repositories.
+Synchronize content rather more often than less often.
+For example, setup a sync plan to synchronize content every day rather than only once a month.
+* Automate the creation and update of sync plans by using a Hammer script or an {ansible-docs-url}[Ansible playbook].
+* Distribute synchronization tasks over several hours to reduce the task load by creating multiple sync plans with *Custom Cron*.
```

Comment:
```suggestion
* Distribute synchronization tasks over several hours to reduce the task load by creating multiple sync plans with the *Custom Cron* tool.
```
Then you can remove the next bullet point.

Reply:
Applied.